Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "DistributionsAD"
 uuid = "ced4e74d-a319-5a8a-b0ac-84af2272839c"
-version = "0.6.4"
+version = "0.6.3"
```

Comment:
I guess there's no need to increase the version number - 0.6.3 will work fine on newer Julia versions as well.